static/templates/tab_bar.hbs

```diff
@@ -13,7 +13,7 @@
     <span class="sub_count" data-toggle="tooltip" title="{{sub_count}} users are subscribed to #{{title}}"><i class="fa fa-user-o"></i>{{formatted_sub_count}}</span>
     {{/if}}
     {{#if rendered_narrow_description}}
-    <span class="narrow_description" data-toggle="tooltip" title="{{rendered_narrow_description}}">{{{rendered_narrow_description}}}</span>
+    <span class="narrow_description" data-toggle="tooltip" title="{{rendered_narrow_description}}">{{rendered_markdown rendered_narrow_description}}</span>
```

Comment:
I think we want to add the `rendered_markdown` CSS class to this span; that will make this use the styling rendering of things like emoji, code blocks, etc. how they look in the message feed...